### PR TITLE
Add Basic ROM Model Setup

### PR DESCRIPTION
**Pull request by RealZiangLiu**
I left the original sphere ROM code commented out in case you have other use for them...

### CODE REVIEW

File: src/multitask_personalization/rom/implicit_mlp.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        self._mlp = nn.Sequential(*layers)
+        self._mlp.to(self.device)
+
+        self._classification_offset = 0.0
+
+    def load(self, ckpt_path=None) -> None:
+        """Load the model from a checkpoint."""
+        if ckpt_path is None:
+            # ckpt_path = "src/multitask_personalization/rom/ckpts/implicit-mlp.model"
```
Comment (tomsilver) [inline src/multitask_personalization/rom/implicit_mlp.py:39]: remove?

File: src/multitask_personalization/rom/implicit_mlp.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            ckpt_path = "src/multitask_personalization/rom/ckpts/implicit-mlp_cpu.pth"
+        # load and convert original model to cpu
```
Comment (tomsilver) [inline src/multitask_personalization/rom/implicit_mlp.py:41]: remove?

File: src/multitask_personalization/rom/models.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        super().__init__()
+        self._rng = rng
+        self._ik_distance_threshold = ik_distance_threshold
+        self._rom_model = MLPROMClassifierTorch(
+            device="cuda" if torch.cuda.is_available() else "cpu"
+        )
+        self._rom_model.load()
+        self._parameter_size = 4
+        self._rom_model_context_parameters = np.array([0.0251, -0.2047, 0.3738, 0.1586])
+        # self._rom_model_context_parameters = np.array(
```
Comment (tomsilver) [inline src/multitask_personalization/rom/models.py:112]: remove?

**Review by tomsilver — APPROVED** [review]
thanks a lot! some tiny requests

File: src/multitask_personalization/rom/models.py (hunk 3 above)
Comment (RealZiangLiu) [reply src/multitask_personalization/rom/models.py:112]: These are sample parameters generated with actual functional scores. A large subspace of the parameter space corresponds to models that predicts infeasible for everything, so I believe these would be helpful for determining the action space bounds.

Comment (tomsilver) [reply src/multitask_personalization/rom/models.py:112]: got it, thanks! can you write that in a comment in the code?

Comment (RealZiangLiu) [reply src/multitask_personalization/rom/models.py]: Done, will merge after CI passes

Comment (RealZiangLiu) [reply src/multitask_personalization/rom/models.py]: Do you have a preference of squash/no squash?

Comment (tomsilver) [reply src/multitask_personalization/rom/models.py]: thanks! yes, please squash 🦶

**Pull request merged**